Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs

```diff
@@ -312,6 +321,55 @@ public static IServiceCollection AddJellyfinApiSwagger(this IServiceCollection s
             });
         }
 
+        /// <summary>
+        /// Sets up the proxy configuration based on the addresses in <paramref name="userList"/>.
+        /// </summary>
+        /// <param name="config">The <see cref="NetworkConfiguration"/> containing the config settings.</param>
+        /// <param name="userList">The string array to parse.</param>
+        /// <param name="options">The <see cref="ForwardedHeadersOptions"/> instance.</param>
+        internal static void ParseList(NetworkConfiguration config, string[] userList, ForwardedHeadersOptions options)
```

Comment:
Maybe rename it to something more descriptive like `AddProxyAddresses` and rename `userList` to allowedProxies. Then I think we're good to go.

Reply:
done